src/locales/cy/translation.json

```diff
@@ -2197,6 +2197,17 @@
           "label": "Rhowch y cod diogelwch",
           "labelSummary": "Dyma’r rhif 6-digid a ddangosir yn eich ap dilysydd"
         }
+      },
+      "2FaBeforeResetPassword": {
+        "header": "Enter a security code from your authenticator app",
```

Comment:
This is put in English on purpose, to indicate that when we will get the welsh translation for this, we will need to replace the text with the translation.

Reply:
noted